Sources/AblyAssetTrackingInternal/AblyWrapper/SDK/AblyCocoaSDK.swift

```diff
@@ -125,8 +125,14 @@ struct AblyCocoaSDKEventListener: AblySDKEventListener {
 public class AblyCocoaSDKRealtimeFactory: AblySDKRealtimeFactory {
     public init() {}
     
-    public func create(withConfiguration configuration: ConnectionConfiguration, logHandler: InternalARTLogHandler) -> AblySDKRealtime {
-        let realtime = ARTRealtime(options: configuration.getClientOptions(logHandler: logHandler, remainPresentForMilliseconds: configuration.remainPresentForMilliseconds))
+    public func create(withConfiguration configuration: ConnectionConfiguration, host: Host?, logHandler: InternalARTLogHandler) -> AblySDKRealtime {
```

Comment:
Most languages would have the `host` parameter last, as it's optional. Should we do the same here? :)

Reply:
Done 🙂